mqtt/mqtt-broker/config/production.json

```diff
@@ -1,14 +1,11 @@
 {
-    "transports": [
-        {
-            "tcp": {
-                "address": "0.0.0.0:1883"
-            }
+    "transports": {
+        "tcp": {
+            "address": "0.0.0.0:1883"
         },
-        {
-            "tls": {
-                "address": "0.0.0.0:8883"
-            }
+        "tls": {
+            "address": "0.0.0.0:8883",
+            "certificate": "broker-production-cert.pfx"
```

Comment:
We might want another name here.

Reply:
actually you need just remove this line at all.